ecospheres/migrations/20250507_1_rename_themes.py

```diff
@@ -0,0 +1,59 @@
+import json
+
+from minicli import cli, run
+
+from ecospheres.api import DatagouvfrAPI
+
+TAG_PREFIX = "ecospheres-theme"
+
+THEMES = [
+    # old, new
+    ("consommer", "mieux-consommer"),
+    ("produire", "mieux-produire"),
+    ("preserver", "mieux-preserver-valoriser-ecosystemes"),
+    ("se-deplacer", "mieux-se-deplacer"),
+    ("se-loger", "mieux-se-loger"),
+    ("se-nourrir", "mieux-se-nourrir"),
+    ("chantiers-transverses", "autre"),
+]
+
+
+def compute_tag(radical: str):
+    return f"{TAG_PREFIX}-{radical}"
+
+
+def migrate_tags(tags: list[str], move: bool = False) -> list[str]:
```

Comment:
handle move to be idempotent

Reply:
Right, good catch...
And while you're at it, `new_tags` should be a set. Shouldn't matter, but clearer intent.